src/index.ts

```diff
@@ -133,26 +133,28 @@ class ServerlessOutputPlugin implements Plugin {
 		);
 	}
 
-	format(exports: Exports, format: string) {
-		switch (format.toLowerCase()) {
-			case "toml":
-				throw new Error("Not implemented yet");
-
-			case "yaml":
-				throw new Error("Not implemented yet");
-
-			case "json":
-				throw new Error("Not implemented yet");
-
-			case "env":
-				return Object.entries(exports)
-					.map(([key, value]) => `${key}=${value}`)
-					.join("\n");
-
-			default:
-				throw new Error(`Format ${format} is not supported`);
-		}
-	}
+	format(exports, format) {
+        switch (format.toLowerCase()) {
+            case "toml":
+                throw new Error("Not implemented yet");
+            case "yaml":
+                return Object.entries(exports)
+                    .map(([key, value]) => `${key}: ${value}`)
+                    .join("\n");
+            case "yml":
+                return Object.entries(exports)
+                    .map(([key, value]) => `${key}: ${value}`)
+                    .join("\n");
```

Comment:
Let's merge these two branches:

```suggestion
            case "yaml":
            case "yml":
                return Object.entries(exports)
                    .map(([key, value]) => `${key}: ${value}`)
                    .join("\n");
```